api/src/main/webapp/WEB-INF/app.yaml.template

```diff
@@ -0,0 +1,30 @@
+service: api
+runtime: java11
+threadsafe: true
+application: "all-of-us-workbench-test"
+env_variables:
+  GAE_PROFILER_MODE: "cpu,heap"
+  JAVA_USER_OPTS: -Djava.util.logging.config.file=webapps/root/WEB-INF/logging.properties
+  SPRING_DATASOURCE_DRIVER_CLASS_NAME=${DB_CONNECTION_STRING}
+  SPRING_DATASOURCE_URL=${DB_CONNECTION_STRING}
+  SPRING_DATASOURCE_USERNAME=${WORKBENCH_DB_USER}
+  SPRING_DATASOURCE_PASSWORD=${WORKBENCH_DB_PASSWORD}
+  CDR_DATASOURCE_URL=${CDR_DB_CONNECTION_STRING}
+  CDR_DATASOURCE_USERNAME=${CDR_DB_USER}
+  CDR_DATASOURCE_PASSWORD=${CDR_DB_PASSWORD}
+error_handlers:
+  - file: server_unavailable.html
+handlers:
```

Comment:
I did not exclude README file because I did not find that syntax. Feel that is not important and we can include.
Otherwise, I may have to refactor this for each file? Like:
handleres:
-url:file1
  blabla:
-url:file2
  blabla:

Reply:
How about: `upload: static/.*\.(js|html|sh)$`, per the docs this is just a regex match (currently we don't have HTML here, but it's not a stretch to expect we might need it here later)